config.go

```diff
@@ -141,3 +145,24 @@ func (c *Config) set(key string, value interface{}) {
 func (c *Config) Get(key string) interface{} {
 	return c.v.Get(key)
 }
+
+// NewFs is just for testing purposes.
+func NewFs(t *testing.T, configBytes []byte, configDir string, f afero.Fs) func() {
+	fs = f
+	filePath := path.Join(configDir, configFileName)
+	require.NoError(t, afero.WriteFile(fs, filePath, configBytes, 0644))
+	lockFile := path.Join(os.TempDir(), filePath+".lock")
+	require.NoError(t, os.MkdirAll(path.Dir(lockFile), 0777))
+	lockFilePath = func(p string) string {
+		return lockFile
+	}
+	return func() {
+		os.Remove(lockFile)
+	}
+}
+
+func FileToBytes(t *testing.T, filePath string) []byte {
+	b, err := ioutil.ReadFile(filePath)
+	require.NoError(t, err)
+	return b
+}
```

Comment:
I would put the test helpers in their own subpackage. Something like "github.com/TheCacophonyProject/go-config/testing`. That way non-test code doesn't end up including things like testify.